packages/ngircd/fix-hardcoded-path.patch

```diff
@@ -0,0 +1,64 @@
+--- ngircd-26/autogen.sh	2021-11-01 11:27:20.660709063 +0700
++++ ngircd-26.mod/autogen.sh	2021-11-01 11:54:22.496709160 +0700
+@@ -68,7 +68,7 @@
+ 	major="$2"
+ 	minor=99
+ 
+-	[ -n "$PREFIX" ] && searchlist="${PREFIX}/$1 ${PREFIX}/bin/$1 $searchlist"
++	[ -n "$TERMUX_PREFIX" ] && searchlist="${TERMUX_PREFIX}/$1 ${TERMUX_PREFIX}/bin/$1 $searchlist"
+ 
+ 	for name in $searchlist; do
+ 		$EXIST "${name}" >/dev/null 2>&1
+@@ -122,10 +122,10 @@
+ [ -z "$EXIST" ] && existlist="type which" || existlist="$EXIST"
+ EXIST=""
+ for t in $existlist; do
+-	$t /bin/ls >/dev/null 2>&1
++	$t ${TERMUX_PREFIX}/bin/ls >/dev/null 2>&1
+ 	if [ $? -eq 0 ]; then
+-		rm -f /tmp/test.$$
+-		$t /tmp/test.$$ >/dev/null 2>&1
++		rm -f ${TERMUX_PREFIX}/tmp/test.$$
++		$t ${TERMUX_PREFIX}/tmp/test.$$ >/dev/null 2>&1
+ 		[ $? -ne 0 ] && EXIST="$t"
+ 	fi
+ 	[ -n "$EXIST" ] && break
+@@ -222,7 +222,7 @@
+ 	# all of them to it.
+ 	NAME=$(grep PACKAGE_STRING= configure | cut -d"'" -f2)
+ 	if [ "$GO" = "1" ]; then
+-		[ -n "$PREFIX" ] && p=" --prefix=$PREFIX" || p=""
++		[ -n "$TERMUX_PREFIX" ] && p=" --prefix=$TERMUX_PREFIX" || p=""
+ 		c="./configure${p}${CONFIGURE_ARGS}"
+ 		echo "Okay, autogen.sh for $NAME done."
+ 		echo "Calling \"$c\" ..."
+--- ngircd-26/config.guess	2020-05-10 22:03:37.000000000 +0700
++++ ngircd-26.mod/config.guess	2021-11-01 14:49:26.912709786 +0700
```

Comment:
The config.guess script is already replaced by build, no need to edit it: https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_replace_guess_scripts.sh

Reply:
I will modify the patch. Thanks for informing me